src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/AcknowledgeServiceCallbackHandlerTest.java

```diff
@@ -14,6 +14,7 @@
 import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
 import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
 import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.handler.callback.user.AcknowledgeServiceCallbackHandler;
```

Comment:
can we also move tests, so this import won't be neccessary?

Reply:
good spot, thought intellij refactor would update these too